fields/civicontributionpages.php

```diff
@@ -34,35 +33,34 @@ class JFormFieldCiviContributionPages extends JFormField
    * Method to get the field input markup.
    *
    * @return  string  The field input markup.
-   * @since  1.6
+   * @since   1.6
    */
-  protected function getInput()
-  {
+  protected function getInput() {
     // Initialize variables.
     $html = array();
 
     // Initiate CiviCRM
-    require_once JPATH_ROOT.'/'.'administrator/components/com_civicrm/civicrm.settings.php';
+    require_once JPATH_ROOT . '/' . 'administrator/components/com_civicrm/civicrm.settings.php';
```

Comment:
why don't you just combine the forward slash with /administrator. no need to break it out. that's an artifact from when we were using the joomla directory separator constant.